feature-assets/src/main/java/io/novafoundation/nova/feature_assets/domain/send/SendInteractor.kt

```diff
@@ -43,14 +43,18 @@ class SendInteractor(
 
             TransferFee(originFee, fees.executionFee)
         } else {
-            val submissionFee = getAssetTransfers(transfer).calculateFee(transfer, coroutineScope = coroutineScope)
             TransferFee(
-                originFee = OriginFee(submissionFee, null),
+                originFee = getOriginFee(transfer, coroutineScope),
                 crossChainFee = null
             )
         }
     }
 
+    suspend fun getOriginFee(transfer: AssetTransfer, coroutineScope: CoroutineScope): OriginFee = withContext(Dispatchers.Default) {
```

Comment:
I dont think we should use `OriginFee` since it also contains informaition about cross chain transfers which is irrelevant for the regular transfer. We can just use `SubmissionFee` returned from `etAssetTransfers(transfer).calculateFee`